apps/gsuite/mail_syncer/tests/apiclient_auth_tests.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.test import TestCase, override_settings
+from mock import patch
+
+from apps.gsuite.auth import build_g_suite_service
+from apps.gsuite.mail_syncer.utils import setup_g_suite_client
+
+
+class GSuiteAPIClientTestCase(TestCase):
+    def setUp(self):
+        self.ow4_gsuite_sync = settings.OW4_GSUITE_SYNC
```

Comment:
I think assigning it like this will modify the original setting too?
For example `self.ow4_gsuite_sync['ENABLED'] = False` 

Not completely sure though.